server/src/api/events/event.service.ts

```diff
@@ -5,38 +5,37 @@ import {pool, response, buildResponse} from '../db';
 export const getActiveEvents = async (): Promise<response> => {
   const myQuery = {
     text: `
-          SELECT eventid id, 
+        SELECT 
+          eventid id, 
           seasonid, 
           eventname title, 
           eventdescription description, 
           active, 
           imageurl image_url, 
           count(eventinstances.eventinstanceid) as numShows
-          FROM events natural join eventinstances
-          GROUP BY events.eventid,
+        FROM events natural join eventinstances
```

Comment:
I think we should avoid using NATURAL JOIN, it makes reading the query more difficult unless you know all the fields of the table. I think an equivalent command would be `FROM eventinstances JOIN events ON eventid` if the schema I'm looking at is current. The commands should also be capitalized for consistency

Reply:
I agree, it's probably a better idea to be explicit about the joins.